test/net/sourceforge/kolmafia/request/EatItemRequestTest.java

```diff
@@ -61,4 +64,44 @@ void milkResponseSetsPreference() {
       assertFalse(Preferences.getBoolean("milkOfMagnesiumActive"));
     }
   }
+
+  @ParameterizedTest
+  @CsvSource({
+    "10991, pizzaOfLegendEaten",
+    "10992, calzoneOfLegendEaten",
+    "11000, deepDishOfLegendEaten"
+  })
+  public void canTrackCookbookbatFoodsSuccess(Integer itemId, String prefname) {
+    assertFalse(Preferences.getBoolean(prefname));
```

Comment:
I would put all of these tests inside something like:
```
    try (withProperty(prefname, false) {
    }
```
so that the pref is restored to its original state after each test finishes.

As is, each of the three tests leaves the property set to true.

Reply:
I assumed the BeforeEach would handle that, but it's good practice to use a cleanups regardless, so I'll do just that.